guide/14-deep-learning/text-classification-workflow.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
```

Comment:
<ol><li>Text classification also known as text tagging or text categorization is the process of assigning tags/labels to <s>an</s> unstructured text.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>fixed</p>